### ISSUE TITLE
LfMerge integration tests with real MongoDB container

### ISSUE THREAD

**Issue by rmunn** [sillsdev/LfMerge#345]
LfMerge integration tests in #342 currently run against a mock Mongo connection. But for real end-to-end testing, we probably want to set up a real MongoDB instance and populate the test project in Mongo, then run the tests that way. We're not talking about setting up an entire local Language Forge, just a Mongo database.

This will be a good follow-up issue once #342 is merged.

**Comment by rmunn** [comment]
To launch a local Mongo, I'd probably use `docker run mongo` with appropriate options (`-d` to detach and run in background, `-n lfmerge-test-db` to have a predictable name, `-p 27017` to forward container port 27017 to a random local port, and so on). Then run `docker port lfmerge-test-db 27017` to find out which random local port docker has found available, and configure LfMerge's Mongo connection to use that port.
